src/page/MyStorePage/components/StoreInfo/StoreInfo.module.scss

```
@@ -20,12 +127,12 @@
     line-height: 47px;
     align-items: center;
     color: #252525;
-    margin-right: 8px;
+    margin-right: 16px;
     font-family: "Noto Sans CJK KR", sans-serif;
   }
 
-  &__keywords {
-    margin-left: 8px;
+  &__small-keywords {
```

이런 클래스명은 엘리먼트보단 modifier로 표현하는 게 적절하다고 생각하는데 어떤가요?
```suggestion
  &__keywords--sm {
```
큰 키워드나 기본적인 사이즈가 있다면 중첩해서 사용하는게 좋아보이구요
```suggestion
  &__keywords {
    // ...
    &--sm {
```

---

keyword의 small과 large의 차이가 `width`밖에 없는거같은데,
글자수에 따라  width를 조정할수있는 속성을 이용하면 하나의 css로 통일하는것도 가능할것같아요

---

width를 auto로 두고 padding을 주어 여백을 살렸습니다! 조언 감사합니다!